ci/scripts/connect-release-init-npm.js

```diff
@@ -85,10 +85,10 @@ const initConnectRelease = () => {
         });
     }
 
-    const PACKAGE_PATH = path.join(ROOT, 'packages', 'connect');
-    const PACKAGE_JSON_PATH = path.join(PACKAGE_PATH, 'package.json');
+    const CONNECT_PACKAGE_PATH = path.join(ROOT, 'packages', 'connect');
+    const CONNECT_PACKAGE_JSON_PATH = path.join(CONNECT_PACKAGE_PATH, 'package.json');
```

Comment:
I changed the name from `PACKAGE_PATH` to `CONNECT_PACKAGE_PATH` because `PACKAGE_PATH` is already used above in this script for each of the packages and this is used here only for connect.